hw7/task1/stringBuilder.js

```diff
@@ -0,0 +1,21 @@
+const {Builder} = require('./builder');
+
+class StringBuilder extends Builder{
+    constructor(data){
+        super(data);
+    }
+    delete(index, count){       
```

Comment:
each method in builder should return `this`. Exept `to` and `build` methods. It used for chains like
``` ts
const builder = new SomeBuilder();
const result = builder.action1().action2().action3().build()
```